lib/index.js

```diff
@@ -418,4 +436,11 @@ client.on('message', message => {
       return { toxicity: NaN, insult: NaN, combined: NaN }
     }
   }
+
+  function findRule(ele,term){
+      if (ele.value.search(term) !== -1 || ele.name.search(term) !== -1){
```

Comment:
One small change is that it should lowercase searched and provided term before searching, so that you !rule discord and !rule Discord would work the same way.

Reply:
done